cmd/iso.go

```
@@ -0,0 +1,203 @@
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// ISO command flags
+	isoSearchPaths []string
```

golangci-lint (1.64.8)

var `isoSearchPaths` is unused

(unused)